qiskit_nature/second_q/algorithms/excited_states_solvers/__init__.py

```diff
@@ -11,22 +11,11 @@
 # that they have been altered from the originals.
 
 """
-Excited State Solving Algorithms (:mod:`qiskit_nature.algorithms.excited_states_solvers`)
+Excited State Solving Algorithms (:mod:`qiskit_nature.second_q.algorithms.excited_states_solvers`)
 =========================================================================================
```

Comment:
If it does, it would be super nice, because I did have to adapt a lot of these underlines manually (maybe this one is just above the threshold of complaint)